File: 116/[for] 敗家的理由.cpp
#include <bits/stdc++.h>
using namespace std;

#define miyuki_is_my_wife ios::sync_with_stdio(false), cin.tie(nullptr)

int main()
{
    miyuki_is_my_wife;

    /*
        水爛
    */
    int n{}, val{}, sum{}, cnt{};
    cin >> n;
    vector<int> arr(n);
    for (auto &i : arr) {
        cin >> i;
    }
    sum = std::accumulate(all(arr), 0);
    cnt = std::count(all(arr), [](const int val) { return val >= 2000 });
//     while (n--) {
//         cin >> val;
//         sum += val;
//         if (val >= 2000) {
//             ++cnt;
//         }
//     }

    if (sum >= 88000 && cnt >= 5) {
        cout << "10888\n";
    } else if (sum >= 88000) {
        cout << "8888\n";
    } else if (sum < 88000 && cnt >= 5) {
        cout << "2000\n";
    } else {
        cout << "0\n";
    }

    return 0;
}

File: 116/[for] 敗家的理由.py
def main():
    n = int(input())
    arr = list(map(int, input().split()))
    
    s = sum(arr)
    c = sum(1 for i in arr if i >= 2000)
    
    if s >= 88000 and c >= 5:
        print("10888")
    elif s >= 88000:
        print("8888")
    elif sum < 88000 and cnt >= 5:
        print("2000")
    else:
        print("0")

if __name__ == "__main__":
    # miyuki is my wife and sister

    main()
